components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -267,6 +274,12 @@ func (backend *ESClient) InsertInspecReport(ctx context.Context, id string, endT
 
 func (backend *ESClient) InsertInspecProfile(ctx context.Context, data *relaxting.ESInspecProfile) error {
 	mapping := mappings.ComplianceProfiles
+	if len(data.Controls) == 0 {
```

Comment:
is it an all or zero thing?  in other words, if we're supposed to have 10 controls, would it be possible to have less than that in a report, yet still more than 0?

Reply:
I've never seen that before. When we get controls from inspec, we get all of them.
Even this situation with 0 controls it's only happening on profiles with run status of skipped/failed.